examples/with-docker-multi-env/docker/development/Dockerfile

```diff
@@ -47,6 +47,5 @@ USER nextjs
 EXPOSE 3000
 
 ENV PORT 3000
-ENV HOSTNAME localhost
 
-CMD ["node", "server.js"]
+CMD HOSTNAME=localhostt node server.js
```

Comment:
Excuse me for interrupting.
Is there a typo?

Reply:
Oops my bad..thank you for the catch! Updated!